### PR TITLE
[test PR]chore: test Spring boot 3.4

### PR DESCRIPTION
**Pull request by ZheSun88**
this is a test PR. so let us dont merge it until a decision has been made from flow team

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -85,7 +85,7 @@
         <failOnMissingWebXml>false</failOnMissingWebXml>
```
Comment (knoobie) [inline pom.xml:85]: OT: I was checking the build to see what created the failures.. it was really hard to spot with all the noise.. you might be interested to include this maven opt to all your builds to reduce the noise from dependency downloads: `-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn`

it removes those lines:

```log


Downloaded from central: https://repo.maven.apache.org/maven2/org/ow2/asm/asm/9.5/asm-9.5.jar (122 kB at 332 kB/s)
Progress (1): 0.9/2.2 MB
Progress (1): 0.9/2.2 MB
Progress (1): 0.9/2.2 MB
Progress (1): 0.9/2.2 MB
Progress (1): 0.9/2.2 MB
Progress (1): 0.9/2.2 MB
Progress (1): 1.0/2.2 MB
Progress (1): 1.0/2.2 MB
Progress (1): 1.0/2.2 MB
Progress (1): 1.0/2.2 MB


```

**Comment by tepi** [comment]
Just updating, got part of the tests fixed (changed security config to comply with new version), but there's still another problem. With the version upgrade `vaadin.blocked-packages` has no effect anymore, everything goes through. Investigating to find the reason.

**Comment by mcollovati** [comment]
Teppo's changes looks good to me.

**Comment by mshabarov** [comment]
I'd propose to merge this pre-release version to get it in use earlier. SPB 3.4 release date is 3 weeks before 24.6 release date, see https://github.com/spring-projects/spring-boot/milestone/353, Nov 21 vs Dec 11, should be aligned very well.

**Review by mshabarov — APPROVED** [review]

**Pull request merged**